app/indexers/concerns/hyku_indexing.rb

```diff
@@ -49,9 +49,14 @@ def extract_full_text(object)
   end
 
   def extract_text_from_plain_text_files(object)
-    return [] if object.members.blank?
-
-    text_file_sets = object.members.select { |fs| fs.file_set? && fs.original_file&.mime_type == 'text/plain' }
+    members = if object.respond_to?(:members)
+                object.members
+              else
+                Hyrax.custom_queries.find_child_file_sets(resource: object)
```

Comment:
this is great, how come we don't just use this all the time and not use `#members`?

Reply:
good question! 🤔

Reply:
I'd have to test it but I think that would work.